.changeset/tame-goats-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/utils": minor
+---
+
+Move vitest matcher to @fuel-ts/utils making it globally accessible across packages. The matcher can now be used throughout the SDK without individual imports and extensions.
```

Comment:
```suggestion
chore: integrate vitest matchers globally
```

Now you have two changeset's, this should match the title and delete the other one (`.changeset/clean-plums-wait.md`)